flexdll.install

```diff
@@ -0,0 +1,14 @@
+share: [
+  "Makefile"
+  "cmdline.ml"
+  "coff.ml"
+  "Compat.ml.in"
+  "create_dll.ml"
+  "default.manifest"
+  "default_amd64.manifest"
+  "flexdll.c"
+  "flexdll.h"
+  "flexdll_initer.c"
+  "reloc.ml"
+  "version.rc"
+]
```

Comment:
Oh! late notice: shouldn’t there be `flexdll.opam` in there, since we’ll now need it to get the version number?

Reply:
(Confession: I didn’t try to build an OCaml compiler with these sources :-)

Reply:
😱 Eek - I hadn't either, I'm very glad you spotted this!

```
C:\Devel>opam pin add flexdll git+https://github.com/dra27/flexdll.git#flexdll-source-packaging
[flexdll.0.43] synchronised (git+https://github.com/dra27/flexdll.git#flexdll-source-packaging)
flexdll is now pinned to git+https://github.com/dra27/flexdll.git#flexdll-source-packaging (version 0.43)

The following actions will be performed:
=== recompile 6 packages
  ↻ base-domains        base          [uses ocaml]
  ↻ base-nnp            base          [uses base-domains]
  ↻ flexdll             0.43 (pinned)
  ↻ ocaml               5.1.1         [uses ocaml-base-compiler]
  ↻ ocaml-base-compiler 5.1.1         [uses flexdll]
  ↻ ocaml-config        3             [uses ocaml-base-compiler]

Proceed with ↻ 6 recompilations? [y/n] y

<><> Processing actions <><><><><><><><><><><><><><><><><><><><><><><><><><>  🐫
⊘ removed   base-nnp.base
⊘ removed   base-domains.base
⊘ removed   ocaml.5.1.1
⊘ removed   ocaml-config.3
⬇ retrieved ocaml-base-compiler.5.1.1  (cached)
⊘ removed   ocaml-base-compiler.5.1.1
⊘ removed   flexdll.0.43
∗ installed flexdll.0.43
[ERROR] The compilation of ocaml-base-compiler.5.1.1 failed at "make -j19".

#=== ERROR while compiling ocaml-base-compiler.5.1.1 ==========================#
# context     2.2.0~beta2~dev | win32/x86_64 |  | git+file://C:/Devel/opam-repository-3#windows-initial
# path        ~\AppData\Local\opam\test-5.1.1\.opam-switch\build\ocaml-base-compiler.5.1.1
# command     ~\AppData\Local\opam\.cygwin\root\bin\make.exe -j19
# exit-code   2
# env-file    ~\AppData\Local\opam\log\ocaml-base-compiler-42868-728b84.env
# output-file ~\AppData\Local\opam\log\ocaml-base-compiler-42868-728b84.out
### output ###
# [...]
# /usr/bin/make -C flexdll-sources MSVC_DETECT=0 OCAML_CONFIG_FILE=../Makefile.config CHAINS=mingw64 ROOTDIR=.. \
#   OCAMLRUN='$(ROOTDIR)/boot/ocamlruns.exe' NATDYNLINK=false \
#   OCAMLOPT='$(OCAMLRUN) $(ROOTDIR)/boot/ocamlc -use-prims ../runtime/primitives -nostdlib -I ../stdlib' \
#   -B flexlink.exe support
# make[3]: Entering directory '/cygdrive/c/Users/DRA/AppData/Local/opam/test-5.1.1/.opam-switch/build/ocaml-base-compiler.5.1.1/flexdll-sources'
# make[3]: *** No rule to make target 'flexdll.opam', needed by 'version.ml'.  Stop.
```